packages/core/src/nuxt.js

```diff
@@ -136,6 +136,15 @@ export default class Nuxt {
     this.readyMessage = null
   }
 
+  isPortInUse(port) {
+    return new Promise((resolve, reject) => {
+      const tester = require('net').createServer()
+        .once('error', err => (err.code === 'EADDRINUSE' ? resolve(true) : reject(err)))
+        .once('listening', () => tester.once('close', () => resolve(false)).close())
```

Comment:
@fureweb-com seems we've got a problem with this line. It's not enough time for the port to be cleared and reused, as the failing tests suggest.

Reply:
Also probably a good idea to follow @pimlie's idea.